.eslintrc

```diff
@@ -41,7 +41,11 @@
     "padded-blocks": ["error", "never"],
     "radix": "error",
     "semi": ["error", "never"],
-    "space-before-function-paren": ["error", "never"],
+    "space-before-function-paren": ["error", {
```

Comment:
this makes it so we can use proper formatting of `async () => {}`. otherwise, the rule forces `async() => {}` which doesn't align with how we usually do things such as `async function() {}`.